tests/beaker_tests/radius_server/radius_server_provider_defaults.rb

```diff
@@ -155,16 +155,18 @@
   # @step [Step] Checks radius_server on agent using switch show cli
   # cmds.
   step 'TestStep :: Check radius_server instance on agent' do
-    # Expected exit_code is 0 since this is a vegas shell cmd.
-    # Flag is set to false to check for presence of RegExp pattern in stdout.
-    cmd_str = get_vshell_cmd('show running-config radius all')
-    on(agent, cmd_str) do
-      search_pattern_in_output(stdout, [/radius-server host 8\.8\.8\.8.* timeout.*$/],
-                               true, self, logger)
-      search_pattern_in_output(stdout, [/radius-server host 8\.8\.8\.8.* retransmit.*$/],
-                               true, self, logger)
-      search_pattern_in_output(stdout, [/radius-server host 8\.8\.8\.8.* key.*$/],
-                               true, self, logger)
+    if platform != 'xrv9k'
+      # Expected exit_code is 0 since this is a vegas shell cmd.
+      # Flag is set to false to check for presence of RegExp pattern in stdout.
+      cmd_str = get_vshell_cmd('show running-config radius all')
```

Comment:
The long-term direction is to move away from any vshell commands in our Beaker tests. Unless you feel that this test step provides significant value, you might consider removing it altogether (or replacing it with a 'puppet resource' check) rather than just excluding XR.